templates/cluster/aws-hosted-cp/templates/k0smotroncontrolplane.yaml

```diff
@@ -55,6 +55,10 @@ spec:
                 - --cluster-cidr={{ first .Values.clusterNetwork.pods.cidrBlocks }}
                 - --allocate-node-cidrs=true
                 - --cluster-name={{ include "cluster.name" . }}
+              cloudConfig:
+                enabled: {{ ne .Values.managementClusterName "" }}
+                global:
+                  KubernetesClusterID: {{ .Values.managementClusterName }}
```

Comment:
And then we can just have `required` if you want to have some guards on that value

Reply:
done!

Reply:
No I mean required in terms of [helm chart itself.](https://helm.sh/docs/chart_template_guide/function_list/#required) 

I'm not sure that schema validation alone will be enough.